### DISCUSSION TITLE
Support for Multiline Input of Conversation Variables

### DISCUSSION THREAD

**Discussion by weilun-enso** [langgenius/dify#10482] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Yes. I have multiple LLM nodes that share a common section called "context". Ideally, I want to put the section in a "constant text" node so that I only need to edit one place for all LLM nodes.

Currently, I have to duplicate the text section multiple times every time some editing is done. Its quite labor intensive (I have 10 LLMs that need to share the section)

The closest thing to this is the conversation variable. However, it does not enable multiline editing for its default value as shown below.
![image](https://github.com/user-attachments/assets/476a7eab-11dc-4eb6-9262-798e06cfeb53)
Note that even the description can be multiline.

I also tried setting the section as input, but it resets every time I close the project. It's also not a good solution.

Alternatively, I think having a "constant" node to store some text to be shared by other nodes is a longer term solution IMO.

### 3. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by sunweilun** [comment]
+1

**Comment by weilun-enso** [comment]
Anyone?
